gnd/src/main/java/com/google/android/gnd/ui/offlinebasemap/OfflineBaseMapsViewModel.java

```diff
@@ -21,41 +21,64 @@
 import androidx.lifecycle.LiveDataReactiveStreams;
 import com.google.android.gnd.model.basemap.OfflineBaseMap;
 import com.google.android.gnd.repository.OfflineBaseMapRepository;
+import com.google.android.gnd.rx.annotations.Cold;
 import com.google.android.gnd.ui.common.AbstractViewModel;
 import com.google.android.gnd.ui.common.Navigator;
 import com.google.common.collect.ImmutableList;
+import io.reactivex.Flowable;
 import javax.inject.Inject;
+import timber.log.Timber;
 
 /**
  * View model for the offline area manager fragment. Handles the current list of downloaded areas.
  */
 public class OfflineBaseMapsViewModel extends AbstractViewModel {
 
-  private LiveData<ImmutableList<OfflineBaseMap>> offlineAreas;
-  private LiveData<Integer> noAreasMessageVisibility;
+  @Cold(terminates = false, errors = false)
```

Comment:
Thinking out loud, I wonder if we shouldn't omit these qualifiers for `LiveData`, since they don't actually carry error or termination states. If anything they would apply to the upstream Flowable. Wdyt?

Reply:
sgtm!